presto-main/etc/catalog/pinot.properties

```diff
@@ -0,0 +1,4 @@
+connector.name=pinot
```

Comment:
This file may not be necessary.

Reply:
Added it here to help testing locally. Is it ok to keep it here?